build.gradle

```
@@ -45,6 +45,9 @@ dependencies {
     implementation 'com.neuronrobotics:java-bowler:3.28.0'
     implementation 'org.apache.commons:commons-csv:1.9.0'
 
+    // Use JavaFX MeshImporter for STL files
+    implementation files('lib/jimStlMeshImporterJFX.jar')
```

It's usually cleaner to let gradle handle and download dependencies, and since this library seems to be available in the maven repository, I suggest to directly use it here
```suggestion
    implementation 'us.ihmc:jimStlMeshImporterJFX:0.7'
```

---

You can then also delete the jar file from the lib directory